tests/ci/run_pytest.py

```diff
@@ -56,24 +61,20 @@ def create_arg_parser():
                     "-m", "not notebooks and not spark and not gpu",
                     "--junitxml=reports/test-unit.xml"])
     '''
-    logger.debug("args.junitxml", args.xmlname)
-    logger.debug("junit=", "--junitxml="+args.xmlname)
-    logger.info('pytest run:',
-                ["pytest",
-                 args.testfolder,
-                 "-m",
-                 args.testmarkers,
-                 "--junitxml="+args.xmlname])
+    logger.debug('args.junitxml {0}'.format(args.xmlname))
+    logger.info('pytest run:  [pytest, {0} "-m" {1} "--junitxml={2}" ]'.format(
```

Comment:
i would suggest building the execution string first then you can use it in the log and the subprocess.run call, that way they'll never get out of sync.

```
pytest_cmd = ['pytest', args.testfolder, '-m', args.testmarkers, '--junitxml={}'.format(args.xmlname)]

logger.info('pytest.run: {}'.format(' '.join(pytest_cmd))
subprocess.run(pytest_cmd)
```

Reply:
done!